zebra/if_netlink.c

```diff
@@ -1527,6 +1527,14 @@ int netlink_link_change(struct nlmsghdr *h, ns_id_t ns_id, int startup)
 		return -1;
 	name = (char *)RTA_DATA(tb[IFLA_IFNAME]);
 
+	/* Must be valid string. */
+	len = RTA_PAYLOAD(tb[IFLA_IFNAME]);
+	if (len == 1 || name[len - 1] != '\0') {
```

Comment:
Would it be safer to check for `len < 2` instead of `len == 1` to also cover the case of `len == 0`?

Reply:
Sure - done!